### ISSUE TITLE
Add existing cluster

### ISSUE THREAD

**Issue by UAnton** [01org/virtual-storage-manager#545]
Hi! I want try install VSM on existing cluster but don't know how do this. In manual I see only clean install VSM and deploy new nodes. 
Thanks!

**Comment by flyingfish007** [comment]
@UAnton If you have an existing ceph cluster, and you have an admin node **without any ceph services as vsm controller node(ceph cluster nodes as vsm agent node)**, now you can start to install vsm. In fact, you can upload the vsm release package to the controller node and follow the clean install VSM. So now you have installed the vsm on your ceph cluster. **Then login in the UI, you can import the existing ceph cluster.**

**Comment by UAnton** [comment]
Install but vsm don't wotk. https://vsm_node01/dashboard/vsm - every time "403" error. VSM work only with openstack?

**Comment by flyingfish007** [comment]
@UAnton If you only want to install controller at first time with parameters "--controller <controller-ip>", you should run commands like "./install.sh -v 2.1 --controller <controller-ip> --prepare".

**Comment by UAnton** [comment]
To made this start on CentOS 7, we must commented some entries in vsm.spec and vsm-dashboard.spec files.
```
#BuildRequires:    python-keystoneclient
#BuildRequires:    python-oslo-config

```

**Comment by flyingfish007** [comment]
Yes, I think you are right. Two packages that you list are not necessary when packing. You can commit and PR. Thanks.
